partner-information.html

```diff
@@ -99,6 +99,17 @@ <h3>Potsdam Institute for Climate Impact Research</h3>
 				</div>
 			</div>
 
+			<div class="row sponsor-information">
+				<div class="col-md-4">
+					<img src="../hackhpi_assets/2018-partners/UFZ_white.jpg" class="img-responsive">
+				</div>
+
+				<div class="col-md-8">
+				<h3>Helmholtz Centre for Environmental Research – UFZ</h3>
```

Comment:
On their website it's `Helmholtz-Centre for Environmental Research - UfZ`, which version should we use?

Reply:
huh good question, would leave it as is as that is extracted from their mission statement

Reply:
Alright.